src/databricks/labs/ucx/progress/install.py

```diff
@@ -52,3 +55,50 @@ def run(self) -> None:
         self._schema_deployer.deploy_table("workflow_runs", WorkflowRun)
         self._schema_deployer.deploy_table("historical", Historical)
         logger.info("Installation completed successfully!")
+
+
+class VerifyProgressTracking:
```

Comment:
@nfx : What do you think of these `Verify...` classes? They are small classes that mainly function as a wrapper around one method

Reply:
ideally we'd merge those

Reply:
> VerifyHasMetastore and VerifyHasCatalog

let's start with merging those once we have cycles for it